pkg/reconciler/inmemorychannel/dispatcher/controller.go

```diff
@@ -109,3 +116,30 @@ func NewController(
 
 	return r.impl
 }
+
+func FilterWithAnnotation(namespaced bool) func(obj interface{}) bool {
```

Comment:
might be able to use this guy that we upstreamed to pkg... https://github.com/knative/pkg/blob/3a272190c3e22b5b9fc5508a8fe255b5a1510389/reconciler/filter.go#L22

Reply:
example usage here: https://github.com/knative/serving/blob/fc69b0f8c1947277e786ce7cfed3651f3a00b610/pkg/reconciler/autoscaling/hpa/controller.go#L59

Reply:
thanks! I was looking for this one!